sql/core/src/test/resources/sql-tests/inputs/try_reflect.sql

```diff
@@ -0,0 +1,19 @@
+-- positive
+SELECT try_reflect("java.util.UUID", "fromString", "a5cf6c42-0c85-418f-af6c-3e4e5b1328f2");
+SELECT try_reflect("java.lang.String", "valueOf", 1);
+SELECT try_reflect("java.lang.Math", "max", 2, 3);
+SELECT try_reflect("java.lang.Math", "min", 2, 3);
+SELECT try_reflect("java.lang.Integer", "valueOf", "10", 16);
+
+-- negative
```

Comment:
does it really matter to put so many similar tests? I'd like to think more about the test coverage:
- runtime errors
- non-existing methods
- non-existing class
- invalid invocation (`try_reflect()` with no arg), since I caught https://github.com/apache/spark/pull/42661/files#r1306863750